qa/rpc-tests/abandonconflict.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python2
```

Comment:
Let's not change these - I don't think there's a interpreter name that works on all distros and is still general, unfortunately. E.g., Ubuntu 16.04 would require an explicit `python2.7` right now.

When the QA tests are ported to Python3 we can just make them all `python3`.